packages/sources/kaiko/src/endpoint/realized-vol.ts

```diff
@@ -0,0 +1,63 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { config } from '../config'
+import overrides from '../config/overrides.json'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { transport } from '../transport/realized-vol'
+
+const RESULT_PATHS = ['realVol1Day', 'realVol7Day', 'realVol30Day']
+const DEFAULT_RESULT_PATH = RESULT_PATHS[2]
+
+export type ResponseData = {
+  [key: string]: number
+}
```

Comment:
Could we type this more strongly, like we did in Coinmetrics?

https://github.com/smartcontractkit/external-adapters-js/blob/main/packages/sources/coinmetrics/src/endpoint/realized-vol.ts#L6-L12

Reply:
Yeah, makes sense. We should also do this in Tiingo (or the FW) then.